src/tools/msc.lua

```diff
@@ -158,6 +192,12 @@
 --
 
 	msc.cxxflags = {
+		cppdialect = {
+			["C++14"] = "/std:c++14",
```

Comment:
How does the cppdialect keyword work currently in VS? I use it in my projects and it seems to work properly. Some other codepath?

Reply:
vs20XX generator handle flags internally.
msc.lua is not used from vs20XX generators.

msc.lua is used for Codelite/Code::Blocks/Ninja (gmakeX doesn't support it) generator with msc toolset.